tools/api/generate_go_protobuf.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+from subprocess import check_output
+from subprocess import call
+
+import glob
+import os
+import shutil
+import sys
+
+# Find the locations of the workspace root and the generated files directory.
+workspace = check_output(['bazel', 'info', 'workspace']).strip()
+bazel_bin = check_output(['bazel', 'info', 'bazel-bin']).strip()
+targets = '@envoy_api//...'
+import_base = 'github.com/envoyproxy/data-plane-api/api'
+output_base = 'go_out'
```

Comment:
Nit: prefer something like `build_go`, we tend to prefix all our temporary build dirs this way.

Reply:
Done